routers/routes/routes.go

```diff
@@ -523,6 +523,10 @@ func RegisterRoutes(m *macaron.Macaron) {
 				return
 			}
 		})
+
+		m.Group("/branches", func() {
+			m.Get("/_new/*", context.RepoRef(), repo.CreateBranch)
```

Comment:
@ethantkoenig not with jquery if only doing with some js timeouts to check if semantic ui dropdown has rendered that link and than binding on click but that is quite dirty that I would want to avoid. Also it could be done using `<a onclick` with bunch of javascript but that code also seems more like hack than good solution